docs/getting-started.md

```diff
@@ -80,99 +84,285 @@ await Kilt.init({ address: YOUR_CHAIN_ADDRESS })
+      attesterSigningKeypair.alg
+    ),
+  },
+}
+
+const { extrinsic, did } = await Kilt.Did.DidUtils.writeDidFromPublicKeys(
+  keystore,
+  attester.address,
+  keys
+)
+
+// The attester must have balance to pay for the transaction to write the newly created full DID on-chain. Submitting the transaction from the extrinsic.
+
+await Kilt.BlockchainUtils.signAndSubmitTx(extrinsic, attester, {
+  reSign: true,
+  resolveOn: Kilt.BlockchainUtils.IS_FINALIZED,
+})
+
+// The resolved full DID
+const fullDid = await Did.DefaultResolver.resolveDoc(did)
+
+console.log('Full DID', fullDid.details)
 
+const fullDid = {
+  keys: {},
+  keyRelationships: {
+    authentication: [
+      'did:kilt:4siJtc4dYq2gPre8Xj6KJcSjVAdi1gmjctUzjf3AwrtNnhvy#0x49d513f193ca2e6401be6bda48a9cfba479b41f4d8b0b68cb516c1229582f68c',
+    ],
+    keyAgreement: [
+      'did:kilt:4siJtc4dYq2gPre8Xj6KJcSjVAdi1gmjctUzjf3AwrtNnhvy#0xdad1085e0941aa6b67488abfeeb1d16cf3f6e8e8453e89fc57113f9486c4f1a7',
+    ],
+    assertionMethod: [
+      'did:kilt:4siJtc4dYq2gPre8Xj6KJcSjVAdi1gmjctUzjf3AwrtNnhvy#0x3b71b98a6bbb5e7c82d9f0c403fdbc9e87c361a3a55d5b6ebe76e1d2f851eb1d',
+    ],
+    capabilityDelegation: [
+      'did:kilt:4siJtc4dYq2gPre8Xj6KJcSjVAdi1gmjctUzjf3AwrtNnhvy#0xc7fb6237d42aec4a04930c9c45fe9b693681f6514766ef4e804ba57facb8aa2b',
+    ],
+    none: [],
+  },
+  didUri: 'did:kilt:4siJtc4dYq2gPre8Xj6KJcSjVAdi1gmjctUzjf3AwrtNnhvy',
+  id: '4siJtc4dYq2gPre8Xj6KJcSjVAdi1gmjctUzjf3AwrtNnhvy',
+  lastTxIndex: '0x0000000000000002',
+}
 ```
-docker run -p 9944:9944 kiltprotocol/mashnet-node:{latest,develop} --dev --ws-port 9944 --ws-external --rpc-external
+
+## 3. How to build and store a Claim Type (CTYPE)
+
+When building a CTYPE, you only need a JSON schema and your public [SS58 address](<https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)>) which you automatically receive when generating an identity.
+
+### 3.1. Building a CTYPE
+
+```typescript
+import Kilt from '@kiltprotocol/sdk-js'
+
+const ctype = Kilt.CType.fromSchema({
+  $schema: 'http://kilt-protocol.org/draft-01/ctype#',
+  title: 'Drivers License',
+  properties: {
+    name: {
+      type: 'string',
+    },
+    age: {
+      type: 'integer',
+    },
+  },
+  type: 'object',
+})
 ```
 
-To store the CTYPE on the blockchain, you have to call:
+### 3.2. Storing a CTYPE
+
+Before you can store the CTYPE on the blockchain, you have to configure your blockchain address and connect to it.
+
+To store the CTYPE on the blockchain with the attester, you have to call:
 
 ```typescript
-// the account behind this mnemonic already has tokens
-const identity = Kilt.Identity.buildFromMnemonic(
-  'receive clutch item involve chaos clutch furnace arrest claw isolate okay together',
-  // using ed25519 as key type because this is how the endowed identity is set up
-  { signingKeyPairType: 'ed25519' }
-)
 const tx = await ctype.store()
 
+const authorizedExtrinsic = await attesterOnChainDid.authorizeExtrinsic(
```

Comment:
Where is `attesterOnChainDid` defined?

Reply:
I have changed `fullDid` to be `attesterOnChainDid`

Reply:
As I said, perhaps `onChainDid` is more confusing than `fullDid`.